mobile/src/Student/screens/SelectTeam/index.js

```diff
@@ -128,7 +129,21 @@ const styles = StyleSheet.create({
   teamIcon: {
     resizeMode: 'contain',
     flex: 1,
-    alignSelf: 'center'
+    alignSelf: 'center',
+    marginTop: 15,
+    //borderWidth: 7,
```

Comment:
nitpick: You should always remove the code instead of comment it.

Reply:
Done in new PR 515